features/step_definitions/journey/order_cards_steps.rb

```diff
@@ -60,9 +62,27 @@
 end
 
 Then(/^the carrier receives an email saying their card order is being printed$/) do
-  # add steps here when last-email functionality is implemented
+  visit(Quke::Quke.config.custom["urls"]["last_email_bo"])
+
+  # Get the text of the last email containing the chosen text.
+  # Take care that this test does not pick up an email from a previous test run.
+  # This test could fail if a similar copy card order has been placed recently.
```

Comment:
NIT: If you check for the content of the email text to include `@registration_number` rather than `"We’re printing your waste carriers registration card"` you should avoid it failing because it picked a similar copy card order